locale/en/corteza-webapp-compose/general.yaml

```diff
@@ -85,6 +85,10 @@ navigation:
   namespace: Namespaces
   module: Modules
   page: Pages
+  single:
```

Comment:
To keep consistent, can you do it like you do in admin (general:label.resource.single/plural)